executor-service/src/main/java/executor/service/service/executor/StepExecutionType.java

```diff
@@ -0,0 +1,32 @@
+package executor.service.service.executor;
+
+import executor.service.service.step.StepExecution;
+import executor.service.service.step.UnsupportedStepExecution;
+import executor.service.service.step.impl.StepExecutionClickCss;
+
+import java.util.Arrays;
+
+public enum StepExecutionType {
+    CLICKCSS(new StepExecutionClickCss()),
+    UNSUPPORTED(new UnsupportedStepExecution());
+    private final StepExecution stepExecution;
+
+    StepExecutionType(
+            final StepExecution stepExecution) {
+        this.stepExecution = stepExecution;
+    }
+
+    public StepExecution getStepExecution() {
+        return this.stepExecution;
+    }
+
+    public static StepExecutionType fromString(
+            final String action) {
+        return Arrays
+                .stream(StepExecutionType.values())
+                .filter(type -> type.name()
+                        .equals(action.toUpperCase()))
+                .findFirst()
+                .orElse(UNSUPPORTED);
```

Comment:
.orElseThrow(() -> new StepExecutionException(String.format("The action %s is not supported.", action));